packages/.eslintrc.json

```diff
@@ -30,13 +31,51 @@
             {
                 "allowExpressions": true
             }
+        ],
+        "lit-a11y/click-events-have-key-events": [
+            "error",
+            {
+                "allowList": [
+                    "sp-button",
+                    "sp-action-button",
+                    "sp-checkbox",
+                    "sp-radio",
+                    "sp-switch",
+                    "sp-menu-item",
+                    "sp-clear-button",
+                    "sp-underlay"
+                ]
+            }
         ]
     },
     "overrides": [
         {
-            "files": ["*.test.ts"],
+            "files": ["*.test.ts", "*.stories.ts", "**/benchmark/*.ts"],
+            "rules": {
+                "spectrum-web-components/document-active-element": ["off"],
+                "lit-a11y/no-autofocus": ["off"],
+                "lit-a11y/tabindex-no-positive": ["off"]
+            }
+        },
+        {
+            "files": ["Dropdown.ts", "SplitButton.ts"],
             "rules": {
-                "spectrum-web-components/document-active-element": ["off"]
+                "lit-a11y/click-events-have-key-events": [
+                    "error",
+                    {
+                        "allowList": [
+                            "sp-button",
+                            "sp-action-button",
+                            "sp-checkbox",
+                            "sp-radio",
+                            "sp-switch",
+                            "sp-menu-item",
+                            "sp-clear-button",
+                            "sp-underlay",
+                            "sp-popover"
```

Comment:
I know in this file this element has interactive children, but not in others. We have the ability to skip this rule in ALL custom elements, but seems like an over step...